### PR TITLE
Not returning an error when trying running eth_getCode against an uni…

### PR DESCRIPTION
**Pull request by lazartravica**
# Description

The eth_getCode JSON RPC call returned an error if the address supplied in the parameter was uninitialized. 
Replicating the behaviour of geth, I have changed it so that it returns a `0x` string.

# Changes include

- [x] Bugfix (non-breaking change that solves an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

# Breaking changes

Please complete this section if any breaking changes have been made, otherwise delete it

# Checklist

- [x] I have assigned this PR to myself
- [x] I have added at least 1 reviewer
- [x] I have tested this code
- [ ] I have updated the README and other relevant documents (guides...)
- [ ] I have added sufficient documentation both in code, as well as in the READMEs

# Additional comments

Please post additional comments in this section if you have them, otherwise delete it

### CODE REVIEW

**Review by Kourin1996 — APPROVED** [review]
LGTM 👏

File: jsonrpc/eth_endpoint.go [hunk 1]
```diff
@@ -462,7 +462,7 @@ func (e *Eth) GetCode(address types.Address, number BlockNumber) (interface{}, e
 
 	acc, err := e.d.store.GetAccount(header.StateRoot, address)
 	if err != nil {
-		return nil, err
+		return "0x", nil
```
Comment (zivkovicmilos) [inline jsonrpc/eth_endpoint.go:465]: We should consider wrapping this in the `arg` types, as we do with other endpoints:
`argUintPtr`, `argBigPtr`...

**Review by zivkovicmilos — APPROVED** [review]
Looks good 💯

**Review by Vuksan — APPROVED** [review]
LGTM.
The only issue I see is that we don't really comply with [the Ethereum specification for the getCode](https://ethereum.org/en/developers/docs/apis/json-rpc/#eth_getcode), but we don't do that for other endpoints as well, so I guess fixing that is a different task.

EDIT: Meaning we don't return result with metadata fields: `id` and `jsonrpc`

**Pull request merged**